tests/unit/test_decorators.py

```diff
@@ -52,6 +52,9 @@ def test_authenticate(self, *args):
         (_, mock_save_token, mock_requests_get, _) = args
         mock_requests_get.return_value = self._expected_response(401)
 
-        self.kytos_auth.authenticate()
+        try:
+            self.kytos_auth.authenticate()
+        except OSError:
+            print("Handle unnecessary OSError")
```

Comment:
We need a message that is more useful to the kytos user here.

Reply:
Which message do you think is more useful? I could think about `Error finding endpoint config request.`

Reply:
I think we need to show a message that:
1. Shows the root cause for the error.
2. Tells the user what she can do about it.

Reply:
In fact, this test is evaluating a result or behavior. A new test is being proposed in https://github.com/kytos/kytos-utils/pull/317